apps/async_requests/sendgrid/newsletter.py

```diff
@@ -28,30 +28,44 @@ def add_subscription(self, contacts):
     def remove_subscription(self, emails):
         pass
 
-    def update_newsletter_preferences(self, profile: Profile, force=False):
+    def sync_newsletter_preferences(self, profile: Profile):
+        """
+        Synchronizes newsletter preferences between Django server and newsletter backend.
+        Does not check for previous state, hence may send redundant data eg. subscribe a user already subscribed
+
+        :param profile: profile to sync preferences
+        :return:
+        """
+        from apps.async_requests.factory import Factory
+
+        # TODO
+        if profile.newsletter_recipient == "TUE":
+            pass
```

Comment:
What is this supposed to do?

Reply:
I was refactoring quite a lot of things and I think I missed this. I'll try making a test for it as well.